libs/sdk-ui-tests/stories/visual-regression/kit/themeWrapper.tsx

```diff
@@ -0,0 +1,36 @@
+// (C) 2020 GoodData Corporation
+import React from "react";
+import { ThemeProvider } from "@gooddata/sdk-ui-theme-provider";
+import { recordedBackend } from "@gooddata/sdk-backend-mockingbird";
+import { ReferenceRecordings } from "@gooddata/reference-workspace";
+import { ITheme } from "@gooddata/sdk-backend-spi";
+
+const workspace = "testWorkspace";
+const theme: ITheme = {
+    palette: {
```

Comment:
there is tooltip missing, which can be then screenshoted as part of Bubble story

Reply:
and what about Dialog?